rust/processor/src/processors/fungible_asset_processor.rs

```diff
@@ -227,6 +250,63 @@ fn insert_current_fungible_asset_balances_query(
     )
 }
 
+fn insert_current_unified_fungible_asset_balances_v1_query(
+    items_to_insert: Vec<CurrentUnifiedFungibleAssetBalance>,
+) -> (
+    impl QueryFragment<Pg> + diesel::query_builder::QueryId + Send,
+    Option<&'static str>,
+) {
+    use schema::current_unified_fungible_asset_balances::dsl::*;
+
+    (
+        diesel::insert_into(schema::current_unified_fungible_asset_balances::table)
+            .values(items_to_insert)
+            .on_conflict(storage_id)
+            .do_update()
+            .set(
+                (
+                    owner_address.eq(excluded(owner_address)),
+                    asset_type.eq(excluded(asset_type)),
+                    coin_type.eq(excluded(coin_type)),
+                    is_frozen.eq(excluded(is_frozen)),
+                    amount_v1.eq(excluded(amount_v1)),
+                    last_transaction_timestamp_v1.eq(excluded(last_transaction_timestamp_v1)),
+                    last_transaction_version_v1.eq(excluded(last_transaction_version_v1)),
+                    inserted_at.eq(excluded(inserted_at)),
+                )
+            ),
+        Some(" WHERE current_unified_fungible_asset_balances.last_transaction_version_v1 IS NULL OR current_unified_fungible_asset_balances.last_transaction_version_v1 <= excluded.last_transaction_version_v1"),
```

Comment:
`current_unified_fungible_asset_balances.last_transaction_version_v1 IS NULL` seems wrong. What if there is a v1 txn and a v2 txn that both get processed. We only want to overwrite the one with largest txn version.

Reply:
it is wrong w/o this.
this only updated the v1 part? wdym

Reply:
Spoke offline with @lightmark. We need to do partial upsert for indexing v1 and v2 balance to handle the case where user sends FA to someone who hasn't migrated their coin to FA yet. In that case the total balance is amount_v1 + amount_v2.